src/ccextractor.c

```diff
@@ -32,6 +32,7 @@ int main(int argc, char *argv[])
 	int ret = 0;
 	enum ccx_stream_mode_enum stream_mode;
 
+	dtvcc_ctx_init(&ccx_dtvcc_ctx);
```

Comment:
can we put ccx_dtvcc_ctx inside library ctx and call this function inside lib_ccx.c